### PR TITLE
fix: Keep using commands' class docstring as help/description

### LINKED ISSUE #64: bug: Docstrings not used without docstring extra
It seems #63 not only removed parsing of docstring, but also the use of docstrings altogether. #62 might have been unclear: I do want to keep documenting my subcommands using class docstrings, I just don't want them to be parsed by a docstring parser and rather just be used raw and converted from Markdown by Rich.

### PR DESCRIPTION
**Pull request by pawamoy**
Fixes #64.

Needs tests :slightly_smiling_face: 
Feel free to push to this branch! `gh pr checkout 65`

### CODE REVIEW

File: src/cappa/command.py [hunk 1]
```diff
@@ -87,17 +87,28 @@ def collect(cls, command: Command[T]) -> Command[T]:
...
+                summary = parsed_help.short_description
+                body = parsed_help.long_description
+            else:
+                doc = inspect.cleandoc(doc)
+                if "\n" in doc:
+                    summary, body = doc.split("\n", 1)
+                    body = body.strip()
+                else:
+                    summary = doc
+                    body = ""
```
Comment (pawamoy) [inline src/cappa/command.py:105]: We could use a `try`/`except ValueError` (unpack) instead.

**Comment by pawamoy** [comment]
Ah, I imitated your branch names, starting with `dc`, but I realize now it probably stands for Dan Cardin :rofl:

File: tests/test_docstring.py [hunk 2]
```diff
@@ -83,3 +83,33 @@ class IncludesDocstring:
...
+
+    @cappa.command()
+    @dataclass
+    class UnparsedDocstring:
+        """Summary.
+        
+        Example:
+
+        Body.
+        """  # noqa: D412
```
Comment (pawamoy) [inline tests/test_docstring.py:106]: I'm not sure the noqa comment is necessary. I had a warning in my editor but it's possible that it's coming from my user configuration.

**Comment by DanCardin** [comment]
> Ah, I imitated your branch names, starting with dc, but I realize now it probably stands for Dan Cardin 🤣
lol!

So I slightly changed the src impl. Then added a test to cover the else branch.

Also slightly modified the test case to utilize rich formatting to prove it was being interpreted that way, since with the extra new line it was causing the lint error and without it was getting inlined by rich formatting.

**Pull request merged**